docs/en/connector-v2/source/Jdbc.md

```diff
@@ -150,6 +150,26 @@ Jdbc {
 }
 ```
 
+redshift:
+
+```
+Jdbc {
+    url = "jdbc:redshift://localhost:5439/testdb?defaultRowFetchSize=1000"
+    driver = "com.amazon.redshift.jdbc42.Driver"
+    connection_check_timeout_sec = 100
+    user = "root"
+    password = "123456"
+    query = "select * from type_bin"
+}
+```
+
+:::tip
+
+defaultRowFetchSize param is very import for JDBC Source to read data from Redshift use Redshift driver.
+If defaultRowFetchSize is not set , the Redshift will return all data to seatunnel source reader and the source may oom.
+
+:::
+
```

Comment:
I think adding defaultRowFetchSize to the appendix URL is enough, and it's not necessary to add it here. What do you think?